documentation/4.0/content/managing-domains/domain-resource.md

```diff
@@ -206,6 +206,16 @@ The elements `serverStartPolicy`, `serverPod` and `serverService` are repeated u
 Elements related to the customization of liveness and readiness probes:
 * See [Liveness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#liveness-probe-customization" >}}) for details about the elements related to liveness probe customization and [Readiness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#readiness-probe-customization" >}}) for details about the elements related to readiness probe customization.
 
+Cluster spec elements:
+
+* `clusterService` - Customization affecting the generation of ClusterIP Service for the WebLogic Server clusters. 
+
+  Sub-elements related to the `clusterService` element:
+
+  * `sessionAffinity`: This is an advanced setting that is only applicable when the `kube-proxy` is running in the non-default proxy modes such as [User space (legacy) proxy mode](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-userspace) and [IPVS proxy mode](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-ipvs). It is used to enable the session affinity based on the client's IP addresses. See the [Virtual IPs and service proxies](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies) for more information. Must be `ClientIP` or `None`. Defaults to `None`.
+
+  **Note:** This setting is not applicable when the `kube-proxy` is running in the default [`iptables` proxy mode](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-iptables). 
+
 {{% notice note %}}
 For additional domain resource attribute reference material, see [Domain resource attribute references](#domain-resource-attribute-references).
```

Comment:
Should this be "For additional domain and cluster ..."?

I assume "#domain-resource-attribute-references" also covers the cluster resource...

Reply:
Changed in [6ce3108](https://github.com/oracle/weblogic-kubernetes-operator/pull/3662/commits/6ce3108e3a015771fa4dd86ab0eea5daf29e3127).